### ISSUE TITLE
The website rendering for license template is missing text inside angle brackets

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-spec#471]
Compare https://github.com/spdx/spdx-spec/blob/development/v2.2.1/chapters/license-matching-guidelines-and-templates.md#b152-template-format- to https://spdx.github.io/spdx-spec/appendix-II-license-matching-guidelines-and-templates/

The website rendering is missing all the text inside the brackets:

```
type: indicates whether the text is replaceable or omitable as per Matching Guideline #2 (“Substantive Text”).
Indicated by <> or...
Indicated by <> and <> respectively.
This field is the first field and is required.
name: name of the field in the template.
This field is unique within each license template.
This field is required.
original: the original text of the rule.
This field is required for a rule type: <>
match: a POSIX extended regular expession (ERE).
This field is required for a rule type: <>
```

perhaps an escaping issue?

**Comment by kestewart** [comment]
@swinslow - can you verify this is still an issue?   otherwise please close.

**Comment by goneall** [comment]
I think this got fixed with @jlovejoy update to the license matching guidelines

**Comment by swinslow** [comment]
This looks fixed for the templating rendering. Though I'm noticing one issue with a bullet-point list in B.3.4 rendering correctly in MarkDown but incorrectly on the website:

MarkDown on GitHub: 
<img width="685" alt="image" src="https://user-images.githubusercontent.com/678569/184152971-5b9b7a87-dd5f-4306-a83d-b63e890d114d.png">

Rendered on Website: 
<img width="591" alt="image" src="https://user-images.githubusercontent.com/678569/184153048-81add744-f145-4873-bce0-28adb22c3a2e.png">

Looks like this is because the MarkDown renderer for the website wants a blank line before and after the bullet point list, but GitHub seems to handle it without that.

I'll submit a PR to fix this momentarily.

**Comment by swinslow** [comment]
I believe that #778 should likely fix this rendering issue, and then I think this overall issue can be closed. Apologies for hijacking this issue with a minor tangent and I can submit a separate issue if you'd prefer.

**Comment by goneall** [comment]
Resolved with #778
